Keas.Mvc/Controllers/Api/SpacesController.cs

```diff
@@ -37,11 +38,17 @@ public async Task<IActionResult> SearchSpaces(string q)
 
             var queryWords = q.Split(" ").Where(w => !string.IsNullOrWhiteSpace(w)).ToArray();
 
-            var space = await _context.Spaces
-                .Where(x => orgIds.Contains(x.OrgId)
-                            && ((!string.IsNullOrWhiteSpace(x.BldgName) && queryWords.Contains(x.BldgName))
-                                || (!string.IsNullOrWhiteSpace(x.RoomName) && queryWords.Contains(x.BldgName))
-                                || (!string.IsNullOrWhiteSpace(x.RoomNumber) && queryWords.Contains(x.BldgName))))
+            var theQuery = _context.Spaces.Where(a => orgIds.Contains(a.OrgId)).AsQueryable();
```

Comment:
Isn't is already Queryable?  Does it actually require AsQueryable?

Reply:
Not sure how that got in there.

Reply:
Oh, I see. Not my branch.